packages/config/config/devices/0x027a/zen05.json

```diff
@@ -0,0 +1,83 @@
+{
+	"manufacturer": "Zooz",
+	"manufacturerId": "0x027a",
+	"label": "ZEN05",
+	"description": "Outdoor Smart Plug",
+	"devices": [
+		{
+			"productType": "0x7000",
+			"productId": "0xb001"
+		}
+	],
+	"firmwareVersion": {
+		"min": "1.1",
+		"max": "255.255"
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"label": "LED Indicator Mode",
+			"valueSize": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Always On",
+					"value": 0
+				},
+				{
+					"label": "Follow Outlet",
+					"value": 1
+				},
+				{
+					"label": "Always Off",
+					"value": 2
+				}
+			]
+		},
+		{
+			"#": "2",
+			"$import": "templates/zooz_template.json#auto_off_timer_duration",
+			"label": "Auto Turn-Off Timer Duration "
+		},
+		{
+			"#": "4",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Auto Turn-On Timer"
+		},
+		{
+			"#": "6",
+			"$import": "~/templates/master_template.json#state_after_power_failure_prev_on_off"
+		},
+		{
+			"#": "7",
+			"label": "Set the brightness level of the LED indicator",
+			"valueSize": 1,
+			"defaultValue": 2,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "high",
+					"value": 0
+				},
+				{
+					"label": "medium",
+					"value": 1
+				},
+				{
+					"label": "low",
+					"value": 2
+				}
+			]
+		},
+		{
+			"#": "8",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Manual Control",
+			"defaultValue": 1
+		}
+	],
+	"compat": {
+		"preserveRootApplicationCCValueIDs": true
```

Comment:
Whats the reason this flag is enabled?

Reply:
Honestly, first time submitting, so tried to mimic what I saw elsewhere where possible.  Could definitely be an error on my part. Did not notice in my testing.

Reply:
I think I resolved it, but did it in an inefficient stupid first-time way.  But ultimately removed the compat and re-commited.